super_team_picker/views/cohorts/all.ejs

```diff
@@ -0,0 +1,21 @@
+<%- include('../partials/header.ejs') %>
+    <% for(cohort of cohorts){ %>
+        <div class="card mx-3 mb-3">
+            <div class="card-body">
+                <div class="card-title d-flex align-items-center">
+                    <img src="<%- cohort.logoUrl%>" class="rounded-circle" alt="cohort logo"
```

Comment:
you can try giving `alt` a real blank image e.g. an image with white background because this will give a broken  image.